client/src/api/azureApi.js

```diff
@@ -20,10 +20,11 @@ export const getSasURL = async (
   userId,
   jobId,
   questionId,
+  interviewId,
   chunkNumber
 ) => {
   return axios.get(
-    `${API_URL}/sas/${userId}/${jobId}/${questionId}/${chunkNumber}`,
+    `${API_URL}/sas/${userId}/${jobId}/${interviewId}/${questionId}/${chunkNumber}`,
```

Comment:
Do we need interviewId? We talked about not needing it because there are no practice interviews. And one interview can only be attempted once.

Reply:
In future if we keep the number of attempts then we need to change the code and as well as we need to change the files on the azure storage also. Having the interview Id won't affect anything now and as well as in future we can maintain that file structure for a long time.